lib/react/server_rendering/exec_js_renderer.rb

```diff
@@ -10,6 +10,7 @@ class ExecJSRenderer
 
       def initialize(options={})
         js_code = options[:code] || raise('Pass `code:` option to instantiate a JS context!')
+        # File.write("./tmp/latest_js_context.js", js_code, mode: "w+")
```

Comment:
@Judahmeek remove comment?

Reply:
I'd rather leave it there & then convert it into a proper trace log statement later on.